src/extensions/renderer/base/load-listeners.js

```diff
@@ -1763,6 +1764,7 @@ BRp.load = function(){
             }
           }
 
+          cy.emit('dragpan');
```

Comment:
On touch, input is imprecise.  The raw touch events that come in can have a delta over time, even when the user didn't intend to drag.  For this reason, you can't kick off gestures until you're reasonably sure that the user intended to perform the gesture.  Here, a threshold delta is used.

In order to make sure that `dragpan` is not erroneously emitted before the threshold is met, `dragpan` should be emitted only directly after each corresponding `panBy()` call.

Reply:
Thank you for looking at it on touch devices. I've made the requested changes.